src/app/containers/MediaPlayer/index.jsx

```diff
@@ -48,27 +50,35 @@ const MediaPlayerContainer = ({ blocks, placeholder }) => {
     return null; // this should be the holding image with an error overlay
   }
 
+  const showPlaceholder = type === 'articles' || embedOverrides.showPlaceholder;
   const placeholderSrc = getPlaceholderSrc(imageUrl);
   const embedSource = embedUrl({
-    requestUrl: `${id}/${versionId}/${lang}`,
-    type: 'articles',
+    requestUrl: `${embedOverrides.id || id}/${versionId}/${lang}`,
```

Comment:
Can you add `const assetId = embedOverrides.id || id` below `type` above please? bit cleaner